app/Exceptions/Handler.php

```diff
@@ -83,16 +85,13 @@ public function render($request, Exception $exception)
                     return response()->json([
                         'message' => 'Error validating requested object.',
                     ], 422);
-                } else {
-                    return response()->json([
-                        'message' => "Internal server error.",
-                    ], 500);
                 }
-                // no break
-            default:
-                return response()->json([
-                    'message' => 'Interval server error.',
-                ], 500);
+                break;
         }
+        $exception = new HttpException(500, 'Whoops!');
```

Comment:
What was the reason to create a new exception here?

Maybe we could just do?
```php
return parent::render($request, $exception);
```

Reply:
Yes I believe this is correct, I'll include it in the next push